Userland/Applications/Calculator/KeypadValue.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Math.h>
+#include <AK/String.h>
+#include <AK/Types.h>
+
+class KeypadValue {
+    friend class Keypad;
+
+public:
+    KeypadValue(i64, u8);
+    KeypadValue(i64);
+
+    //helper lambda for KeypadValue's operators should have access to its private members
```

Comment:
Please leave one space to start the comment, and properly capitalise the contents of the comment.